src/app/components/Article/index.jsx

```diff
@@ -34,14 +39,16 @@ class Article extends Component {
   }
 
   render() {
-    const { headline } = this.state;
+    const { data } = this.state;
+    const { title } = data;
```

Comment:
If you destructure `model` here too, you can pass it to [MainContent](https://github.com/bbc/simorgh/pull/247/files#diff-5f296fd8afe33a8853c3f2655172c257R51), rather than calling `data.model` [here](https://github.com/bbc/simorgh/pull/247/files#diff-89fcabca58d80c9d5cba58347724d5ceR27)